lib/utils/background_push.dart

```diff
@@ -69,7 +72,7 @@ class BackgroundPush {
 
   final pendingTests = <String, Completer<void>>{};
 
-  final dynamic fcmSharedIsolate = FcmSharedIsolate();
+  final FcmSharedIsolate fcmSharedIsolate = FcmSharedIsolate();
```

Comment:
Do you know the reason why it is dynamic and nullable?

Reply:
no, why is it need ?

Reply:
because it's final and the type is clearly `FcmSharedIsolate(); `, so i don't think we need it to be dynamic

Reply:
rolled back